datalad/resources/sshserver_prepare_for_publish.sh

```diff
@@ -30,6 +30,11 @@ cd "$ROOT_PATH"
 mkdir -p "$COL_REPO_NAME"
 git -C "$COL_REPO_NAME" init $git_init_options
 printf "DATALAD_COLLECTION_REPO_%s: init DATALAD_END\n" "$COL_REPO_NAME"
+git -C "$COL_REPO_NAME" update-server-info
+printf "DATALAD_COLLECTION_REPO_%s: update-server-info DATALAD_END\n" "$COL_REPO_NAME"
+mv $COL_REPO_NAME/.git/hooks/post-update.sample $COL_REPO_NAME/.git/hooks/post-update
```

Comment:
ok... I will then just merge and will add a TODO note so it would be easier to catch it later on